CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [1.1.2] - 2017-08-29
```

Comment:
generally it is best to leave the version bump to maintainers as there is no guarantee on response times or merge order.

Reply:
Ah gotcha, sorry first time making a contribute to a public project.  I was trying to follow the guides which states to update the change log, I'll make sure to wait for response in the future :)

Reply:
Totally understandable and welcome to open source!

Ya we need to really write up better guidelines, we want a changelog entry (and we are slightly opinionated flavor of keepachangelog) but leaving the actual version bumping to the maintainers for the above reason.